stdlib/public/Darwin/Accelerate/vImage_Buffer.swift

```diff
@@ -72,6 +72,34 @@ extension vImage_Buffer {
     
     /// Initializes a vImage buffer of a specified size.
     ///
+    /// - Parameter size: The size of the buffer.
+    /// - Parameter bitsPerPixel: The number of bits in a pixel of image data.
+    ///
+    /// - Returns: An initialized vImage buffer.
+    public init(size: CGSize,
```

Comment:
This needs its own `@available` attribute, or it needs to be added to a new extension that has the correct availability. (It inherits `macOS 10.15`, `iOS 13`, etc. from its current context, which is wrong.)

Reply:
Hi, this is fixed now.